openshift/frontend.bc.yaml

```diff
@@ -44,7 +44,7 @@ objects:
       openshift.io/generated-by: OpenShiftNewBuild
     name: "${REPO_NAME}-frontend-${JOB_NAME}"
   spec:
-    completionDeadlineSeconds: 600
+    completionDeadlineSeconds: 900
```

Comment:
We really need to investigate at some point why the build is taking more than 10 minutes to build.

Reply:
Yup